python/xorbits/pandas/tests/test_accessors.py

```diff
@@ -88,7 +89,7 @@ def test_cls_docstring():
     )
 
 
-def test_obj_docstring(setup, dummy_str_series, dummy_dt_series):
+def test_obj_docstring(setup, dummy_str_series, dummy_dt_series, dummy_df):
```

Comment:
Besides obj docstring, you also need to test if the docstring works for class object.

Like,
```
assert xpd.DataFrame.loc.__doc__ is not None and xpd.DataFrame.loc.__doc__.endswith("This docstring was copied from ...")
```